### PR TITLE
Adds a fax machine to requisitions

### PR DESCRIPTION
**Pull request by L0pz**
# About the pull request

Adds a fax machine to requisitions

# Explain why it's good for the game

I've always found it weird how an entire department doesn't have a singular fax machine. Especially since req can use them to ask for unique equipment, which they currently have to do via the public fax machine In the meeting hall.

# Testing Photographs and Procedure
<details>
<summary>

image

</summary>

</details>

# Changelog
:cl:
add: Added a fax machine to requisitions. It's a girl, by the way.
/:cl:

### CODE REVIEW

**Review by efzapa — COMMENTED** [review]
the `desc` vars need quotes around them, that's what's causing it to fail

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 1]
```diff
@@ -9046,7 +9046,11 @@
 "blw" = (
 /obj/structure/surface/table/almayer,
 /obj/item/tool/lighter,
-/obj/structure/machinery/faxmachine/uscm,
+/obj/structure/machinery/faxmachine/uscm{
+	desc = The beloved fax machine of the Requisitions department. Be sure to make a convincing argument when you ask high command for a crate of lunge mines.;
```
Comment (Nanu308) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:9050]: quotation marks required for text strings. Repeat for other errors. Also no need to reference lunge mines.

**Review by Nanu308 — CHANGES_REQUESTED** [review]

**Comment by DexterDude** [comment]
I don't think you removing the SEA fax machine was intentional, you should probably fix that.

**Comment by L0pz** [comment]
> I don't think you removing the SEA fax machine was intentional, you should probably fix that.

Wait shit I thought I had fixed that

**Review by Nanu308 — APPROVED** [review]
Le fine

**Pull request merged**